buildlib/azure-pipelines.yml

```diff
@@ -10,33 +10,65 @@ resources:
     - container: centos7
       image: ucfconsort.azurecr.io/ucx/centos7:1
       endpoint: ucfconsort_registry
+    - container: fedora
+      image: ucfconsort.azurecr.io/ucx/fedora:30
```

Comment:
The version here should not be :30, but :1 - the sequence number should reflect the requirement of this .yml file. Ie if you change the image and start running new commands you have to bump the sequence.

This is important otherwise the CI goes funny on stable branches and when there are different PRs on different bases.